qemu/tests/cfg/hv_enforce_cpuid.cfg

```diff
@@ -0,0 +1,21 @@
+- hv_enforce_cpuid:
+    only Linux
+    type = hv_enforce_cpuid
+    modprobe_cmd = "modprobe msr"
+    rdmsr_cmd = "rdmsr 0x40000002"
+    create_image = yes
+    remove_image = yes
+    kill_vm_gracefully = yes
```

Comment:
Not a must IMO, vm.destroy() can make sure the previous vm can be destroyed.
this option seems to be used for preparation of starting a vm